### ISSUE TITLE
Fail to connect pypi.org when upgrade

### ISSUE THREAD

**Issue by tsukiazuma** [dfir-iris/iris-web#376]
Hi,

I follow step to step:
```
git checkout v2.3.3
docker-compose down
git pull 

git checkout v2.3.7 
docker compose build --no-cache 
docker compose up
```

But when run: `docker compose build --no-cache`, it error: `WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fb689e8f0a0>, 'Connection to pypi.org timed out. (connect timeout=15)')': /simple/pip-date/`

**Comment by whikernel** [comment]
Hi @tsukiazuma 

It looks like your docker daemon are not able to reach internet. Are you using a proxy?

**Comment by tsukiazuma** [comment]
Yeah, I use proxy. But other servers using that proxy still connect normally
And I tried remove proxy -> get error connect with docker images
